project_gtd/i18n/it.po

```diff
@@ -407,7 +410,6 @@ msgstr "o"
 #~ msgid "This Month"
 #~ msgstr "Questo mese"
 
-#, python-format
```

Comment:
All of these strings are commented, so they're useless. Maybe a good idea would be to regenerate italian file, syncing previously all the terms.